core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -570,6 +570,30 @@ private static String toSql(RelNode root, SqlDialect dialect) {
     assertThat(toSql(root), notNullValue());
   }
 
+  /** Test case for
```

Comment:
Do we need a stress test case? For example, a query with 1000 - 3000 OR conditions (as described in the JIRA)?

Reply:
Yes.

Reply:
Just updated a stress test case. see `testStack`.